### ISSUE TITLE
TypeError: only integer scalar arrays can be converted to a scalar index

### ISSUE THREAD

**Issue by dabasmoti** [qdrant/qdrant-client#439]
I am trying to create a collection with sparse vectors 
I am using `mmh3.hash(token, signed=False)` as hashmap.
and I get the error above.

## Steps to Reproduce
```
from qdrant_client import QdrantClient
from qdrant_client import models

client = QdrantClient(":memory:")
COLLECTION_NAME = "bm25"

client.recreate_collection(
    collection_name=COLLECTION_NAME,
    vectors_config={},
    sparse_vectors_config={"text": models.SparseVectorParams()}
)


client.upsert(
collection_name=COLLECTION_NAME,
points=[
    models.PointStruct(
        id=1,
        payload={},  # Add any additional payload if necessary
        vector={
            "text": models.SparseVector(
        indices=[1479876206, 696618268],
        values=[0.5,0.6]
            )
        },
    )
],
)
```

**Comment by agourlay** [comment]
I took the liberty to transfer the issue to the qdrant Python client repository.

**Comment by joein** [comment]
Hi @dabasmoti 

We have a bug on our side.
In case you need to use local mode and don't want to wait for the release, you can sort indices and corresponding values before vector creation, e.g.:

OK:
```
SparseVector(
    indices=[0, 5],
    values=[0.3, 0.2]
)
```

ERROR:
```
SparseVector(
    indices=[5, 0],
    values=[0.2, 0.3]
)
```

**Comment by joein** [comment]
available as of `qdrant-client==1.7.1`
